crates/sui-security-watchdog/src/lib.rs

```diff
@@ -33,8 +31,9 @@ pub struct SecurityWatchdogConfig {
     pub sf_username: Option<String>,
     #[clap(long, default_value = None, global = true)]
     pub sf_role: Option<String>,
-    #[clap(long, default_value = None, global = true)]
-    pub sf_password: Option<String>,
+
+    pub sf_password: String,
```

Comment:
should we still keep them as `Option<String>`

Reply:
I wonder otherwise how `let mut config = SecurityWatchdogConfig::parse();` will work. maybe it'll just use empty string as default value for these two fields?

Reply:
Oh clap seems to be confused by this: `Usage: sui-security-watchdog --pd-wallet-monitoring-service-id <PD_WALLET_MONITORING_SERVICE_ID> --config <CONFIG> <SF_PASSWORD> <PD_API_KEY>`

I might just move these out of the clap configs and pass them into the func's directly